tests/mlmodel_sklearn/test_cluster_models.py

```diff
@@ -19,10 +19,10 @@
 )
 
 from newrelic.api.background_task import background_task
```

Comment:
These changes are in another PR so let's just remove the sklearn changes from this PR.